test/components/toggle_component_test.rb

```diff
@@ -1,12 +1,23 @@
 # frozen_string_literal: true
 
 require "test_helper"
-
 class ToggleComponentTest < ViewComponent::TestCase
-  def test_component_renders_something_useful
-    # assert_equal(
-    #   %(<span>Hello, components!</span>),
-    #   render_inline(ToggleComponent.new(message: "Hello, components!")).css("span").to_html
-    # )
+  def setup
+    @form = mock()
+    @form.expects(:hidden_field).with("Field", value: "Default Value", data: { toggle_target: "hiddenField" })
+  end
+
+  def test_component_renders_something_useful_with_description
+    toggle_component = render_inline(ToggleComponent.new(label: "Label", form: @form, field: "Field", default_value: "Default Value", description: "Description"))
+
+    assert_selector("#availability-label", text: "Label")
+    assert_selector("#availability-description", text: "Description")
+  end
+
+  def test_component_renders_something_useful_without_description
+    toggle_component = render_inline(ToggleComponent.new(label: "Label", form: @form, field: "Field", default_value: "Default Value"))
+
+    assert_selector("#availability-label", text: "Label")
+    refute_selector("#availability-description")
```

Comment:
refute_selector is checking if the expected HTML is not rendered from the component. In this case, a description should not be rendered because one wasn't entered and it is not required.